app/controllers/manager/users_controller.rb

```diff
@@ -0,0 +1,53 @@
+class Manager::UsersController < ApplicationController
+  before_action :require_login
+  before_action :find_user, only: %i(show update)
+  before_action :paginate_members, only: %i(index update)
+  before_action{check_role? :manager}
```

Comment:
gộp lại dòng này với dòng 2

Reply:
vâng ạ

Reply:
Em không biết cho lên kiểu gì. Nó toàn báo syntax error anh ạ.